tests/test_loop.cpp

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
@smartalecH, should this file be included in `make check`?

Reply:
No, this was just for my personal debugging. Didn't realize it was added.